runtime/src/lib.rs

```diff
@@ -179,10 +179,11 @@ fn enter_idle(drivers: &mut Drivers) {
 }
 
 fn human_readable_command(bytes: &[u8]) -> Option<&str> {
-    if bytes.len() != 4 || bytes.iter().any(|c| !c.is_ascii_alphanumeric()) {
-        None
+    if bytes.len() == 4 || bytes.iter().all(|c| c.is_ascii_alphanumeric()) {
```

Comment:
Should `||` be `&&`?

Reply:
Thanks! Forgot to invert that when I inverted the rest.